src/pages/kimjinpyeong/Login/LoginJinpyeong.js

```diff
@@ -1,7 +1,86 @@
-import React from 'react';
+import './LoginJinpyeong.scss';
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
 
 const LoginJinpyeong = () => {
-  return <div>Login</div>;
+  const [id, setId] = useState('');
+  const [pw, setPw] = useState('');
+  const [verifyLogin, setVerifyLogin] = useState(false);
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    if (id.length > 5 && id.includes('@') && pw.length > 5) {
+      setVerifyLogin(true);
+    } else {
+      setVerifyLogin(false);
+    }
+  }, [id, pw]);
+
+  const saveUserId = e => {
+    setId(e.target.value);
+  };
+
+  const saveUserPw = e => {
+    setPw(e.target.value);
+  };
+
+  const Login = e => {
+    e.preventDefault();
+
+    if (verifyLogin) {
+      navigate('/main-jinpyeong');
+    }
+  };
+
+  return (
+    <main className="login">
```

Comment:
```suggestion
    <main className="loginJinpyeong">
```
scss를 사용할 때, wecode의 컨벤션은 가장 부모 태그에 컴포넌트의 이름과 동일하게 구현하는 것입니다.

Reply:
넵! 알겠습니다.

Reply:
수정 완료했습니다.